test/GPv2Settlement/ComputeTradeExecutions.t.sol

```diff
@@ -0,0 +1,192 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.8.26;
+
+import {Vm} from "forge-std/Test.sol";
+
+import {GPv2Settlement} from "src/contracts/GPv2Settlement.sol";
+import {GPv2Order, IERC20} from "src/contracts/libraries/GPv2Order.sol";
+import {GPv2Signing} from "src/contracts/mixins/GPv2Signing.sol";
+
+import {GPv2Transfer, Helper, SettlementEncoder} from "./Helper.sol";
+
+import {Order} from "test/libraries/Order.sol";
+import {Registry, TokenRegistry} from "test/libraries/encoders/TokenRegistry.sol";
+
+abstract contract BaseComputeTradeExecutions is Helper {
+    using TokenRegistry for TokenRegistry.State;
+    using TokenRegistry for Registry;
+    using SettlementEncoder for SettlementEncoder.State;
+
+    IERC20 private sellToken;
+    IERC20 private buyToken;
+
+    uint256 internal executedAmount = 10 ether;
+    uint256 internal sellPrice = 1;
+    uint256 internal buyPrice = 2;
+
+    function setUp() public virtual override {
+        super.setUp();
+
+        sellToken = IERC20(makeAddr("ComputeTradeExecutions: sellToken"));
+        buyToken = IERC20(makeAddr("ComputeTradeExecutions: buyToken"));
+    }
+
+    function partialOrder() internal view returns (GPv2Order.Data memory) {
```

Comment:
Nit: it used to be a partial order, but now it's a full-fledged order. Maybe something like `defaultOrder` would be clearer?

Reply:
Done